src/api/app/models/issue.rb

```diff
@@ -4,6 +4,7 @@ class Issue < ApplicationRecord
   class NotFoundError < APIError
     setup 'issue_not_found', 404, 'Issue not found'
   end
+  class InvalidName < APIError; end
```

Comment:
please move all the error classes to `app/models/issue/errors.rb` as we have for the others?

Reply:
a concern for 4 lines? Isn't that a little over the top?

Reply:
all errors classes. Can be more in the future. It's matter of code organization and structure.

Reply:
Done. Issue errors were moved to its own class. Thanks!